README.md

```diff
@@ -1 +1,107 @@
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- - [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+# 📖 <a name="about-project">Hello Word</a>
+> In this project we wanna just great personne who will see our page  We call this project Greating We built this project just by using html and css
+Hello Word is an exercise 
+## 🛠 Built With <a name="built-with">HTML, CSS and Linter</a>
+### Tech Stack <a name="tech-stack"></a>
+> In this project we use just one page html and his document css to make it pretty
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">index.html</a></li>
+    <li><a href="#">styles.css</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+### Key Features <a name="key-features">Hello Microverse</a>
+> We have just one title :
+- Add the HTML for the structure of the project and Add the heading hello microverse
+
+<!-- LIVE DEMO -->
+## 🚀 Live Demo <a name="live-demo"></a>
+> 
+- [Live Demo Link](Coming soon...)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+## 💻 Getting Started <a name="getting-started"></a>
+> To do the same with this project you should create one html file, add in h1 title, next create a css file, make a style and link the css file in html
+
+### Setup
+Clone this repository to your desired folder:
+git clone https://github.com/francksefu/Hello-Word.git
+
+### Usage
+To run the project, open the html folder in your navigator
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- AUTHORS -->
+## 👥 Authors <a name="authors">Franck Sefu Kalunga</a>
+
+👤 Franck Sefu Kalunga
+- GitHub: [@githubhandle](https://github.com/francksefu)
+- Twitter: [@twitterhandle](https://twitter.com/franck_sefu)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/franck-sefu-884705254/)
+👤 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- FUTURE FEATURES -->
+## 🔭 Future Features <a name="future-features">Make it dynamic</a>
+> Describe 1 - 3 features you will add to the project.
+- we will add, in a future, one js file to make this proect dynamic
+- We will add more content
+- 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- CONTRIBUTING -->
+## 🤝 Contributing <a name="contributing"></a>
+Contributions, issues, and feature requests are welcome!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- SUPPORT -->
+## ⭐️ Show your support <a name="support"></a>
+> I love work, solve problems, and I have big dreams, maybe in future I will add some AI in this project, that will be possible if you give me strength
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- ACKNOWLEDGEMENTS -->
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+> I wanna give thank to Microverse
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+This project is [MIT](./LICENSE) licensed.
+
```

Comment:
- [ ] **_[OPTIONAL]_** This requirement is marked optional but highly recommended. Please make sure to include a LICENSE in your project. Microverse suggests you use the MIT one. Follow the [official docs](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to achieve this! 🤓

- [ ] Public repositories on GitHub are often used to share open-source software. For your repository to truly be open source, you'll need to license it so that others are free to use, change, and distribute the software. 💪

Reply:
The LICENSE link is broken. not working.